From 860d6e158dbb581c3aabc6a20ae8d83b325bffd8 Mon Sep 17 00:00:00 2001
From: Wei Liu <wei.liu2@citrix.com>
Date: Fri, 12 Apr 2019 11:03:25 +0100
Subject: [PATCH] xl: handle PVH type in apply_global_affinity_masks again

A call site in create_domain can call it with PVH type. That site was
missed during the review of 48dab9767.

Reinstate PVH type in the switch.

Reported-by: Julien Grall <julien.grall@arm.com>
Signed-off-by: Wei Liu <wei.liu2@citrix.com>
diff --git a/tools/xl/xl_vcpu.c b/tools/xl/xl_vcpu.c
index ef99513..051dc12 100644
--- a/tools/xl/xl_vcpu.c
+++ b/tools/xl/xl_vcpu.c
@@ -78,6 +78,7 @@ void apply_global_affinity_masks(libxl_domain_type type,
 
     switch (type) {
     case LIBXL_DOMAIN_TYPE_HVM:
+    case LIBXL_DOMAIN_TYPE_PVH:
         type_mask = &global_hvm_affinity_mask;
         break;
     case LIBXL_DOMAIN_TYPE_PV:
